pkg/controllers/v1alpha1/databackup/implement.go

```diff
@@ -332,15 +348,49 @@ func (r *DataBackupReconcilerImplement) reconcileExecutingDataBackup(ctx reconci
 // generateDataBackupValueFile builds a DataBackupValueFile by extracted specifications from the given DataBackup, and
 // marshals the DataBackup to a temporary yaml file where stores values that'll be used by fluid dataBackup helm chart
 func (r *DataBackupReconcilerImplement) generateDataBackupValueFile(ctx reconcileRequestContext, masterPod *v1.Pod) (valueFileName string, err error) {
+	log := ctx.Log.WithName("generateDataBackupValueFile")
 	databackup := ctx.DataBackup
 	nodeName, ip, rpcPort := utils.GetAddressOfMaster(masterPod)
 
-	imageName, imageTag := docker.GetWorkerImage(r.Client, databackup.Spec.Dataset, "alluxio", databackup.Namespace)
+	targetDataset, err := utils.GetDataset(r.Client, databackup.Spec.Dataset, databackup.Namespace)
+	if err != nil {
+		return "", err
+	}
+
+	index, boundedRuntime := utils.GetRuntimeByCategory(targetDataset.Status.Runtimes, common.AccelerateCategory)
+	if index == -1 {
+		return "", errors.New("bounded runtime with Accelerate Category is not found on the target dataset")
+	}
+
+	var imageName, imageTag, javaEnv, runtimeType, imageEnv, defaultImage string
+	switch boundedRuntime.Type {
+	case common.ALLUXIO_RUNTIME:
+		imageName, imageTag = docker.GetWorkerImage(r.Client, databackup.Spec.Dataset, "alluxio", databackup.Namespace)
+		javaEnv = "-Dalluxio.master.hostname=" + ip + " -Dalluxio.master.rpc.port=" + strconv.Itoa(int(rpcPort))
+		runtimeType = "alluxio"
```

Comment:
can we use `common.ALLUXIO_RUNTIME` as runtimeType ?

Reply:
lgtm